src/components/global/Button.tsx

```diff
@@ -0,0 +1,115 @@
+import { css } from '@emotion/react';
+import styled from '@emotion/styled';
+import THEME from '@src/constants/Colors';
+import WIDTHS from '@src/constants/Widths';
+import { Size, Variant } from '@src/types/Button.type';
+import React, { ButtonHTMLAttributes } from 'react';
+import MoonLoader from 'react-spinners/MoonLoader';
+
+export interface ButtonProps extends ButtonHTMLAttributes<HTMLButtonElement> {
+  variant: Variant;
+  size: Size;
+  rounder: boolean;
+  isLoading?: boolean;
+}
+
+const Button = ({ children, onClick, isLoading, variant, size, rounder }: ButtonProps) => {
+  return (
+    <StyledButton onClick={onClick} variant={variant} size={size} rounder={rounder}>
+      {/* To be fixed */}
```

Comment:
어떤 내용이 fix가 되어야 되는지 알려주면 좋을 것 같아요!

```suggestion
{/* FIXME:  ...*/}
```